src/main/java/seedu/address/model/HistoryManager.java

```diff
@@ -59,17 +59,22 @@ public void pushRecord(MutatorCommand command, AddressBook addressBook) {
      * Reverts history to the state just before the specified record.
      *
      * @param record the record to revert to
+     * @return list of records popped with the first popped record at index 0
      * @throws NoSuchElementException if the specified record does not exist in the history
      */
-    public void popRecordsTo(HistoryRecord record) throws NoSuchElementException {
+    public List<HistoryRecord> popRecordsTo(HistoryRecord record) throws NoSuchElementException {
         int index = history.indexOf(record);
         if (index == -1) {
```

Comment:
Can avoid magic numbers and instead have a `public static final int` attribute in this class with value `-1` assigned to it